### ISSUE TITLE
DataBase Schema Configuration

### ISSUE THREAD

**Issue by lakakenny** [clun/ff4j#104]
Hello,

It's possible make possible to configure table names.
For example to give prefix and suffix.

Thanks,

**Comment by clun** [comment]
Hi, the table are already prefixed with "FF4J_" at the moment.

I can add this evolution (because I do not use JPA).
It will be an attribute to set to the FeatureStoreJdbc.

Is it b what you would like ?

**Comment by lakakenny** [comment]
Hi, in job we need to use ff4j. But in database schema all table have a
functional prefix and suffix.

Thank you.

**Comment by clun** [comment]
Will be available in 1.5 (in extremis), much more work than i expected. I will work for PropertyStore, FeatureStore and EventRepository for both core JDBC and spring JDBC stores.

By default the _queryBuilder_ will use `FF4_` as table prefix but you can override this way
```
FeatureStoreSpringJdbc jdbcStore = new FeatureStoreSpringJdbc();
jdbcStore.setQueryBuilder(new JdbcQueryBuilder("T_FF4J_", "_01"));
jdbcStore.setDataSource(db);
``` 
Please find a full sample here
https://github.com/clun/ff4j/blob/master/ff4j-store-springjdbc/src/test/java/org/ff4j/test/store/SpringJdbcFeatureStorePrefixTest.java
